### PR TITLE
CryptX added priority as optional parameter and new coin

### PR DESCRIPTION
**Pull request by khidesha**

### CODE REVIEW

**Review by pvyhnal-generalbytes — CHANGES_REQUESTED** [review]

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/cryptx/v2/CryptXWallet.java [hunk 1]
```diff
@@ -129,6 +148,13 @@ public BigDecimal getCryptoBalance(String cryptoCurrency) {
         return null;
     }
 
+    public String getAPICryptocurrency(String cryptoCurrency) {
+        if (cryptoCurrency.equalsIgnoreCase(CryptoCurrency.USDT.getCode())) {
+            return CryptoCurrency.ETH.getCode();
+        }
```
[the remark this answers is not in the document]
Comment (khidesha) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/cryptx/v2/CryptXWallet.java:154]: USDT (Teaser) is an erc20 token and it's on ethereum blockchain. That's why for some cases cryptX wallet API needs be called with "eth" in code.

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 2]
```diff
@@ -80,7 +82,8 @@
                 <cryptocurrency>BTC</cryptocurrency>
```
[the remark this answers is not in the document]
Comment (khidesha) [reply server_extensions_extra/src/main/resources/batm-extensions.xml:82]: Sorry, I can't understand. What do you mean under the tag?

[the remark this answers is not in the document]
Comment (pvyhnal-generalbytes) [reply server_extensions_extra/src/main/resources/batm-extensions.xml:82]: I meant the param tag, sorry.

`<param name="priority" />`

[the remark this answers is not in the document]
Comment (khidesha) [reply server_extensions_extra/src/main/resources/batm-extensions.xml:82]: Ok, can you confirm that all issues are corrected on our side

[the remark this answers is not in the document]
Comment (pvyhnal-generalbytes) [reply server_extensions_extra/src/main/resources/batm-extensions.xml:82]: yes, just please add this tag also to the cryptxnoforward

**Comment by generalbytes** [comment]
We are waiting for suggested changes to be implemented.

**Comment by hungry-programmer** [comment]
All requested changes have been made. Please review again.

**Review by pvyhnal-generalbytes — APPROVED** [review]

**Pull request merged**